packages/components/src/validated-form-controls/test/control-with-error.tsx

```diff
@@ -282,4 +282,57 @@ describe( 'ControlWithError', () => {
 			expect( onSubmit ).not.toHaveBeenCalled();
 		} );
 	} );
+
+	describe( 'Focus behavior', () => {
+		it( 'should focus the first error in the form', async () => {
+			const user = userEvent.setup();
+			render(
+				<form>
+					<ValidatedInputControl label="Text1" required />
+					<ValidatedInputControl label="Text2" required />
+					<button type="submit">Submit</button>
+				</form>
+			);
+
+			await user.click(
+				screen.getByRole( 'button', { name: 'Submit' } )
+			);
+
+			expect(
+				screen.getByRole( 'textbox', { name: /^Text1/ } )
```

Comment:
Nit: Should we expect that the name would be _exactly_ `Text1` ?

```suggestion
				screen.getByRole( 'textbox', { name: 'Text1' } )
```

Reply:
This is just side-stepping the detail that the `required` prop adds a `(Required)` suffix to the labels. Kind of wanted to exclude that detail from these tests.